RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate E2ESaveYourPasswordView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- When is your first time in the app or after resetting your password, will shows up the blue banner wrote Save your Encryption Password
- Then you click on it and will appear a modal. And this is the migrated view

## Screenshots

## Types of changes




## Checklist



## Further comments
